packages/components/src/toolbar/toolbar/index.tsx

```diff
@@ -16,7 +16,18 @@ import deprecated from '@wordpress/deprecated';
 import ToolbarGroup from '../toolbar-group';
 import ToolbarContainer from './toolbar-container';
 import type { ToolbarProps } from './types';
-import type { WordPressComponentProps } from '../../ui/context';
+import {
+	WordPressComponentProps,
+	ContextSystemProvider,
+} from '../../ui/context';
+
+const CONTEXT_SYSTEM_VALUE = {
```

Comment:
I had already left a note a few lines below, but let me know in case you don't think that's clear enough.

Setting this variable now will also allow us to replace the custom [block settings dropdown ](https://github.com/WordPress/gutenberg/blob/057f64b076b5f9a711e566d724fc6eb5c4228aa0/packages/block-editor/src/components/block-settings-menu/index.js) — using the new `DropdownMenu` component in that file should inherit the context set in the `Toolbar` component

Reply:
My suggestion was to prepend it with "TODO" since that implies a future action, vs when there's no TODO, it's not immediately clear to the person that this needs to be changed.

Reply:
Got it! Added a comment in https://github.com/WordPress/gutenberg/pull/51097/commits/0887983f16602f12fb6a7b79730cc0ebfbedf846, let me know in case you would word it differently.

Reply:
All good, thank you 🙌